Add correlating logs blog (#1769)
opensearch-project/project-website: _authors/prabarch.markdown

---
name: Prabhakar Sithanandam
short_name: prabarch
photo: '/assets/media/authors/prabarch.png'
---

> **Prabhakar Sithanandam** is a Principal Engineer in AWS OpenSearch Service. He spends most of his time tinkering OpenSearch engine to make it more performant and optimized for various use cases

Reviewer: ```suggestion
**Prabhakar Sithanandam** is a Principal Engineer at AWS focusing on OpenSearch. He spends most of his time tinkering with the OpenSearch engine to make it more performant and optimized for various use cases.
```
